src/features/login/LoginOverlay.vue

```diff
@@ -1,32 +1,37 @@
 <template>
-  <simple-spinner v-if="currentUserState.loading" />
+  <simple-spinner v-if="loading" />
   <template v-else>
     <slot v-if="initialized" :onLogout="onLogout"></slot>
     <dialog ref="dialogRef" @keyup.escape.prevent @keydown.escape.prevent>
-      <a
-        :href="redirectUrl"
-        target="_blank"
-        @click="onLinkClick"
-        @keydown.enter="onLinkClick"
-        >Je sessie is verlopen. Klik in het scherm om opnieuw in te loggen. Als
-        je dit binnen {{ loginTimeoutInSeconds }} seconden doet, verlies je geen
-        werk.</a
-      >
+      <div>
+        <a
+          :href="redirectUrl"
+          target="_blank"
+          @click="onLinkClick"
+          @keydown.enter="onLinkClick"
+          >Je sessie is verlopen. Klik in het scherm om opnieuw in te loggen.
```

Comment:
Niet het hele scherm is meer klikbaar, alleen de tekst. Dus dan moeten we denk ik de tekst aanpassen(klik hier bv).